### PR TITLE
[site] Added checkbox in reques access form

### PR DESCRIPTION
**Pull request by ToyTox**

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/request_access_form.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <div id="request_access_intro" data-header-form>
   <h3 class="text text_h3">{{ site.data.modal['request-access']['header'][page.lang]['title']}}</h3>
   <p class="text">{{ site.data.modal['request-access']['header'][page.lang]['text']}}</p>
-  <form class="form" action="https://license.deckhouse.io/api/license/request" id="request_access_form" data-form>
+<!--  <form class="form" action="https://license.deckhouse.io/api/license/request" id="request_access_form" data-form>-->
```
Comment (z9r5) [inline docs/site/_includes/request_access_form.html:4]:
```suggestion
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Pull request merged**